samvera/hyku
# add-google-analytics

Changes proposed in this pull request:
- Adding documentation to the README
- Adds edits to the analytics.yaml file to be inline with Hyrax's
- Add the web and worker script
- Update the Dockerfile to call the web script
- Adding the env variables to each deployment values file & .env
GOOGLE_ANALYTICS_ID env variable is [listed in 1Password](https://start.1password.com/open/i?a=LTLZ652TT5H5FHMYMASSH7PIXM&v=huuakin4bu4xanlhktv42qheam&i=2jqkpzsupzfu3ozifme2lec5de&h=scientist.1password.com).

Ref ticket: https://github.com/scientist-softserv/palni-palci/issues/339

Testing Instructions:
- Navigate to a current tenant and see the ga areas of the app (Dashboard/Reports/etc) is working with no errors (if applicable)
- Create a new tenant and navigate to that tenant and see the ga areas of the app (Dashboard/Reports/etc) is working with no errors

Images:
<img width="1151" alt="Screenshot 2023-01-09 at 19 56 57" src="https://user-images.githubusercontent.com/63515648/211459839-40f75989-a95e-4b82-91e1-f5f6a4ef73fb.png">
<img width="1148" alt="Screenshot 2023-01-09 at 19 56 15" src="https://user-images.githubusercontent.com/63515648/211459844-5a8b2f3a-8c9f-4e8d-8e5a-609e1d911af6.png">
<img width="1153" alt="Screenshot 2023-01-09 at 19 55 41" src="https://user-images.githubusercontent.com/63515648/211459846-a77ef446-f364-4be0-8001-a387d60c5d43.png">

@samvera/hyku-code-reviewers
